src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  width: 100px;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-around;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    box-sizing: border-box;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: left;
+
+    &:nth-child(-n + 4) {
```

Comment:
Consider using css from the previous task

Reply:
Previous mentor told me to do that change, it seems to have no difference, must I change it or just "consider"? In what case I sd consider to change it?